src/model/job/dbt-job.ts

```diff
@@ -10,7 +10,7 @@ export type DBTJobConstructorParams = {
   name: string;
 };
 
-export class DBTJobConstructor extends Job {
+export class DBTJob extends Job {
```

Comment:
cleanup naming convention inconsistency

Reply:
Hypothetically, should we still support clients using `DBTJobConstructor`?

```suggestion
export class DBTJob extends Job {
  ...
 }
 
// Untested: 
const DBTJobConstructor = DBTJob;
export DBTJobConstructor;
```

Realistically, it doesn't look like our only client of this is using that class, so this probably doesn't matter. Also, a library upgrade wouldn't be forced upon them, so they could make this small change if they needed to upgrade, right?

Reply:
Nevermind.

Reply:
Yeah, I think we make this a major version bump to be safe and in the release notes we can document the differences.